api/envoy/api/v2/listener.proto

```diff
@@ -195,6 +195,7 @@ message Listener {
   google.protobuf.UInt32Value tcp_fast_open_queue_length = 12;
 
   // Specifies the intended direction of the traffic relative to the local Envoy.
+  // This property is required on Windows for listeners using the original destination filter.
```

Comment:
doc nit: ref link to original destination docs here and below

Reply:
fixed in https://github.com/envoyproxy/envoy/pull/14753/commits/f7698962ac3e21e673937b4e3723f171b87ae8ea